roachtest: npgsql failed

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12737311?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12737311?buildTab=artifacts#/npgsql) on master @ [14733120c08ba1a0a050bb5442c3311090206566](https://github.com/cockroachdb/cockroach/commits/14733120c08ba1a0a050bb5442c3311090206566):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v24.1.0-alpha.00000000-dev-14733120c08ba1a0a050bb5442c3311090206566
Tests run against npgsql v7.0.2
3281 Total Tests Run
2619 tests passed
662 tests failed
248 tests skipped
60 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).Unreferenced_positional_parameter_works -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_metamorphicBuild=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12737311/npgsql/1700306331643/1700306652526)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12867503?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12867503?buildTab=artifacts#/npgsql) on master @ [ba6423f916dc4006bb590c786cf53ad2b0213417](https://github.com/cockroachdb/cockroach/commits/ba6423f916dc4006bb590c786cf53ad2b0213417):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v24.1.0-alpha.00000000-dev-ba6423f916dc4006bb590c786cf53ad2b0213417
Tests run against npgsql v7.0.2
3281 Total Tests Run
2619 tests passed
662 tests failed
248 tests skipped
60 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CopyTests(Multiplexing).Undefined_table_throws -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_metamorphicBuild=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12867503/npgsql/1701172071949/1701172390218)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/npgsql_blocklist.go
+++ b/pkg/cmd/roachtest/tests/npgsql_blocklist.go
@@ -684,6 +684,7 @@ var npgsqlIgnoreList = blocklist{
 	`Npgsql.Tests.CommandTests(Multiplexing).SingleQuery`:                                             "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Statement_mapped_output_parameters(Default)`:             "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).TableDirect`:                                             "flaky",
+	`Npgsql.Tests.CommandTests(Multiplexing).Unreferenced_positional_parameter_works`:                 "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Use_across_connection_change(NotPrepared)`:               "flaky",
 	`Npgsql.Tests.CommandTests(NonMultiplexing).Cached_command_clears_parameters_placeholder_type`:    "flaky",
 	`Npgsql.Tests.CommandTests(NonMultiplexing).CloseConnection_with_exception`:                       "flaky",

--- a/pkg/cmd/roachtest/tests/ruby_pg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/ruby_pg_blocklist.go
@@ -202,4 +202,6 @@ var rubyPGBlocklist = blocklist{
 var rubyPGIgnorelist = blocklist{
 	`PG::Connection OS thread support Connection.new shouldn't block a second thread`:                             "flaky",
 	`running with sync_* methods PG::Connection consume_input should raise ConnectionBad for a closed connection`: "flaky",
+	`running with sync_* methods PG::Connection OS thread support Connection.new shouldn't block a second thread`: "flaky",
+	`running with sync_* methods PG::Connection handles server close while asynchronous connect`:                  "flaky",
 }

